kinetic/distribution.yaml

```diff
@@ -1617,6 +1617,22 @@ repositories:
       url: https://github.com/introlab/find-object.git
       version: kinetic-devel
     status: maintained
+  follow_waypoints:
+    doc:
+      type: git
+      url: https://github.com/danielsnider/follow_waypoints.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/danielsnider/follow_waypoints-release.git
+      version: 0.2.0-0
+    source:
+      test_pull_requests: true
```

Comment:
Yes I think so, see screenshot: 
![image](https://user-images.githubusercontent.com/1737560/28003069-898fdb96-6508-11e7-8a06-8b175194efc4.png)
Thanks for helping :-) maybe I do need some since it's my first package.